Makefile

```diff
@@ -1,6 +1,6 @@
 CONCORD_BFT_DOCKER_REPO:=concordbft/
 CONCORD_BFT_DOCKER_IMAGE:=concord-bft
-CONCORD_BFT_DOCKER_IMAGE_VERSION:=0.9
+CONCORD_BFT_DOCKER_IMAGE_VERSION:=1.0
```

Comment:
Andrew mentioned this but I'm marking it explicitly. Our docker image should probably stay below 1.0 as long as concord-bft isn't 1.0. `0.10` should work.